bower.json

```diff
@@ -18,12 +18,16 @@
     "templates"
   ],
   "dependencies": {
-    "angular": "1.2.18",
+    "angular": "~1.3.0",
     "bootstrap-select": "1.5.4",
-    "common-style": "https://github.com/Rise-Vision/common-style.git",
-    "jquery": "1.11.0",
+    "jquery": "~2.1.4",
+    "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.14",
```

Comment:
@donnapep Curious, is upgrading to such a high version going to give any version conflicts in widget-settings-ui-components and our widgets? Will it still be fine in those to override this to be a lower version?